dashboard/app/models/pd/workshop_daily_survey.rb

```diff
@@ -62,22 +65,47 @@ def set_day_from_form_id
       :day
     )
 
+    LOCAL_CATEGORY = 'local'
```

Comment:
It's unfortunate this is called `local` when it really should be summer workshops (as opposed to academic year). This is leftover from when we initially wrote this for just local thinking TeacherCon would be separate, but then we extended this to TeacherCon. 

It might be worth the effort of changing it now that we're adding more categories, to something like `summer`. Note this would require adding (temporary) duplicate form ids in chef for the new category.

Reply:
Now that teachercons are over, are all future forms in this category local summer workshops? I's still not an ideal name, but that would be less bad.

Reply:
Good point. Local is less bad in that light, but still they're all going to be local. The distinction is summer vs. academic year. I still support changing it to summer.

Reply:
Lets call it `local_summer`